src/styles/blocks/card.scss

```
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 19px;
+  font-size: $font-size;
+  line-height: $line-height;
+  color: $main-color;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__header {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 0 4px;
```

reset default margin for `h2` and `p` the same as you did with body to avoid reseting this value each time
```suggestion
    margin-bottom: 4px;
```